website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,679 @@
+              <div className="bg-gray-100 dark:bg-charcoal-700 rounded-lg p-4 h-[50vh] lg:h-[75vh] overflow-y-auto">
+                {readmeState.readmeLoading ? (
+                  <div className="flex items-center justify-center py-8">
+                    <div className="animate-spin rounded-full h-8 w-8 border-b-2 border-purple-600"></div>
+                    <span className="ml-2 text-gray-600 dark:text-gray-300">Loading documentation...</span>
+                  </div>
+                ) : readmeState.readmeError ? (
+                  <div className="text-red-600 dark:text-red-400 text-center py-4">{readmeState.readmeError}</div>
+                ) : (
+                  <div
+                    className="prose prose-sm max-w-none text-gray-800 dark:text-gray-200 prose-headings:text-charcoal-300 dark:prose-headings:text-white prose-a:text-purple-600 dark:prose-a:text-purple-400 prose-code:text-purple-600 dark:prose-code:text-purple-400 prose-pre:bg-gray-200 dark:prose-pre:bg-charcoal-600 prose-pre:text-gray-800 dark:prose-pre:text-gray-200"
+                    dangerouslySetInnerHTML={{ __html: formatMarkdownAsHtml(readmeState.readmeContent) }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)